docs/CHANGELOG.md

```diff
@@ -11,12 +11,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added support for the Nethermind Ethereum client.
 - Added support for batch sending telemetry to the ingress server to improve performance.
+- Added disk rotating logs.
```

Comment:
Might be nice to add a bit more detail here, this is a great feature. Let's explain to node operators how this helps both them and us.